### PR TITLE
visualize network access, i.e. queried point to snapped waypoint

### PR DESCRIPTION
**Pull request by oblonski**
looks like this

Screenshot 2023-03-28 at 20 18 27

### CODE REVIEW

**Comment by oblonski** [comment]
Screenshot 2023-03-28 at 20 20 35

File: src/layers/UsePathsLayer.tsx [hunk 1]
```diff
@@ -73,6 +78,58 @@ function addUnselectedPathsLayer(map: Map, paths: Path[]) {
     map.addInteraction(select)
 }
 
+function createBezierLineString(start: number[], end: number[]):LineString {
+    var curvePoints = [];
```
Comment (easbar) [inline src/layers/UsePathsLayer.tsx:82]: You can always use `const`, unless you get a (Typescript) compiler error. Then you can use `let`. `var` is basically as thing of the past.

File: src/layers/UsePathsLayer.tsx [hunk 2]
```diff
@@ -73,6 +78,58 @@ function addUnselectedPathsLayer(map: Map, paths: Path[]) {
...
+            lineDash: [1, 10],
+            lineCap: 'round',
+            lineJoin: 'round'
+        }),
+    })
+    var layer = new VectorLayer({
+        source: new VectorSource()
+    });
+    layer.setStyle(style);
+    for(let i=0;i<selectedPath.snapped_waypoints.coordinates.length;i++){
```
Comment (easbar) [inline src/layers/UsePathsLayer.tsx:123]: For consistent formatting you can run `npm run format` or integrate the prettier configuration with your IDE or text editor.

**Comment by easbar** [comment]
> looks like this

Looks very nice :)

File: src/layers/UsePathsLayer.tsx (hunk 1 above)
Comment (oblonski) [reply src/layers/UsePathsLayer.tsx]: done

File: src/layers/UsePathsLayer.tsx (hunk 2 above)
Comment (oblonski) [reply src/layers/UsePathsLayer.tsx]: Oh ja, thanks for the hint. Just did it.

Comment (easbar) [reply src/layers/UsePathsLayer.tsx]: Du kannst das natürlich auch mergen, oder soll ich?

Comment (oblonski) [reply src/layers/UsePathsLayer.tsx]: Mach lieber du. Ich bin hier nur Gast.

Comment (oblonski) [reply src/layers/UsePathsLayer.tsx]: Danke @easbar

Comment (easbar) [reply src/layers/UsePathsLayer.tsx]: Ein willkommener Gast. Danke dir!

**Pull request merged**